src/actions/BringMoveSwap.ts

```diff
@@ -163,12 +163,13 @@ class BringMoveSwap implements Action {
               ? edits
               : edits.filter(({ isSource }) => !isSource);
 
-          const editSelectionInfos = edits.map(({ originalTarget }) =>
-            getSelectionInfo(
-              editor.document,
-              originalTarget.contentSelection,
-              DecorationRangeBehavior.OpenOpen
-            )
+          const editSelectionInfos = edits.map(
+            ({ edit: { range }, originalTarget }) =>
+              getSelectionInfo(
+                editor.document,
+                selectionFromRange(originalTarget.isReversed, range),
```

Comment:
This should always use the edit range.  Before, the edit range and `originalTarget.contentSelection` were equal, so it didn't matter, but now that the edit range is at the start of the line for line targets, the range updater wasn't actually updating the edit range 😳.  It appears that the way you implemented `updateRange` allowed you to work around this bug, but when I tried to simplify things, the bug became apparent